gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
And this file. Anything dealing with gradle besides build.gradle is auto-generated at this point.